sources/CapstoneDisassembler.cpp

```diff
@@ -95,7 +95,7 @@ void PLH::CapstoneDisassembler::setDisplacementFields(PLH::Instruction& inst, co
 
 				// skip rex prefix
 			    (hasGroup(capInst, x86_insn_group::X86_GRP_JUMP) && inst.size() >= 3 && inst.getBytes().at(1) == 0xff && inst.getBytes().at(2) == 0x25) ||
-				(hasGroup(capInst, x86_insn_group::X86_GRP_CALL) && inst.size() >= 3 && inst.getBytes().at(1) == 0xff && inst.getBytes().at(2) == 0x25)
+				(hasGroup(capInst, x86_insn_group::X86_GRP_CALL) && inst.size() >= 3 && inst.getBytes().at(1) == 0xff && inst.getBytes().at(2) == 0x15)
```

Comment:
this change breaks indirect jmps, which is what 0x25 encodes. Please add an additional case, do not remove this one

Reply:
After the change, the code looks like:

```c++
if ((hasGroup(capInst, x86_insn_group::X86_GRP_JUMP) && inst.size() >= 2 && inst.getBytes().at(0) == 0xff && inst.getBytes().at(1) == 0x25) ||
    (hasGroup(capInst, x86_insn_group::X86_GRP_CALL) && inst.size() >= 2 && inst.getBytes().at(0) == 0xff && inst.getBytes().at(1) == 0x15) ||

    // skip rex prefix
    (hasGroup(capInst, x86_insn_group::X86_GRP_JUMP) && inst.size() >= 3 && inst.getBytes().at(1) == 0xff && inst.getBytes().at(2) == 0x25) ||
    (hasGroup(capInst, x86_insn_group::X86_GRP_CALL) && inst.size() >= 3 && inst.getBytes().at(1) == 0xff && inst.getBytes().at(2) == 0x15) //<== my fix
  ){
   inst.setIndirect(true);

  ...

```

Sorry I am not sure why this change will breaks indirect JMPs? in my case, the machine code is: 

```asm
00007FFA69484264 | 48:FF15 BDEB0500         | call qword ptr ds:[<&RtlSetIoCompletionCallback>] 
00007FFA6948426B | 0F1F4400 00              | nop dword ptr ds:[rax+rax],eax
```
If I rollback my fix from 0x15 to 0x25, then the SIGSEGV error occurs,  because the content of dest-holder will be the memory address of qword ( 0x00007FFA6948426B + 0x0005EBBD) pointing to call target, instead of the target itself (RtlSetIoCompletionCallback); the indirect flag is not set properly by the unpatched code.

Reply:
I am sorry I read the patch diff wrong, you are correct